raiden/raiden_service.py

```diff
@@ -92,6 +92,8 @@
 )
 
 log = slogging.get_logger(__name__)  # pylint: disable=invalid-name
+# register filelock logger
+filelock.logger = slogging.get_logger('filelock')
```

Comment:
It's really annoying that this is necessary. IMO this really is a bug in slogging and should be fixed there. &lt;/rambly-non-actionable-comment&gt;